en/flight_controller/pixhawk4.md

```diff
@@ -37,58 +37,66 @@ Pixhawk<sup>&reg;</sup> 4 is an advanced autopilot designed and made in collabor
 
 
-## Build Instructions
+## Purchase
 
-`make px4fmu-v5_default upload`
+Order from [Holybro](https://shop.holybro.com/pixhawk-4beta-launch_p1089.html).
 
 
-## Purchase
-Order from [Holybro](https://shop.holybro.com/pixhawk-4beta-launch_p1089.html)
+## Connectors
 
+![Pixhawk 4 connectors](../../assets/flight_controller/pixhawk4/pixhawk4-connectors.jpg)
 
-## Connectors
-> **Note** The SBUS RC and PPM RC ports are for RC receivers only and provide power. NEVER connect any servos, power supplies or batteries to it 
-or to the receiver connected to it.
+> **Warning** The **DSM/SBUS RC** and **PPM RC** ports are for RC receivers only. These are powered! NEVER connect any servos, power supplies or batteries (or to any connected receiver).
 
-![Pixhawk 4 connectors](../../assets/hardware/hardware-pixhawk4-connectors.png)
 
 ## Pinouts
 
-Download Pixhawk 4 pinouts from [here]().
+Download *Pixhawk 4* pinouts from [here]( ).
 <!-- link to holybro (https://github.com/PX4/Hardware/blob/master/FMUv5/Pixhawk4-Pinouts.pdf)-->
 
+
 ## Voltage Ratings
 
-Pixhawk 4 can be triple-redundant on the power supply if three power sources are supplied. The three rails are: Power module inputs 1 and 2 and 
-USB input.
+*Pixhawk 4* can be triple-redundant on the power supply if three power sources are supplied. The three power rails are: **POWER1**, **POWER2** and **USB**.
+
+> **Note** The output power rails **FMU PWM OUT** and **I/O PWM OUT** (0V to 36V) do not power the flight controller board (and are not powered by it). You must supply power to one of **POWER1**, **POWER2** or **USB** or the board will be unpowered. 
 
 **Normal Operation Maximum Ratings**
 
 Under these conditions all power sources will be used in this order to power the system:
-1. Power module inputs 1 and 2  (4.9V to 5.5V)
-1. USB power input (4.75V to 5.25V)
-
-> **Note** **FMU PWM OUT** and **I/O PWM OUT** (0V to 36V) input voltage does not power autopilot, autopilot will be unpowered if power module input is not present.
+1. **POWER1** and **POWER2** inputs (4.9V to 5.5V)
+1. **USB** input (4.75V to 5.25V)
 
 **Absolute Maximum Ratings**
 
 Under these conditions the system will not draw any power (will not be operational), but will remain intact.
-1. Power module inputs 1 and 2 (operational range 4.1V to 5.7V, 0V to 10V undamaged)
-1. USB power input (operational range 4.1V to 5.7V, 0V to 6V undamaged)
-1. Servo input: VDD_SERVO pin of FMU PWM OUT and I/O PWM OUT (0V to 42V undamaged)
+1. **POWER1** and **POWER2** inputs (operational range 4.1V to 5.7V, 0V to 10V undamaged)
+1. **USB** input (operational range 4.1V to 5.7V, 0V to 6V undamaged)
+1. Servo input: VDD_SERVO pin of **FMU PWM OUT** and **I/O PWM OUT** (0V to 42V undamaged)
 
 
-### Debug Port
+## Assembly/Setup 
+
+The [Pixhawk 4 Wiring Quick Start](../assembly/quick_start_pixhawk4.md) provides instructions on how to assemble required/important peripherals including GPS, Power Management Board etc.
+
+
+## Building Firmware
```

Comment:
FYI this is near end because most people will not need to build firmware who are reading the User Guide.